JavaScript/blog.js

```diff
@@ -0,0 +1,63 @@
+blogEntryStore = [];
+count=1;
+
+function Blog(title,category,author,authorUrl,publishedOn,body){
```

Comment:
you can DRY (don't repeat yourself) this up by passing in an item, and then assigning this.title = item.title. Just make sure to change in the rawData loop as well.